css/src/atomics/flex.scss

```diff
@@ -0,0 +1,423 @@
+}
+
+.justify-content-space-around {
+	justify-content: space-around !important;
+}
+
+.justify-content-space-evenly {
+	justify-content: space-evenly !important;
+}
+
+//Responsive Classes used in docs-ui
+
+.align-items-center-tablet {
+	@include tablet {
+		align-items: center !important;
+	}
+}
+
+.justify-content-center-tablet {
+	@include tablet {
+		justify-content: center !important;
+	}
+}
+
+.justify-content-end-tablet {
+	@include tablet {
+		justify-content: flex-end !important;
+	}
+}
+
+.justify-content-end-desktop {
+	@include desktop {
+		justify-content: flex-end !important;
+	}
+}
+
+// Other suggested Useful responsive classes for tablet
+
+.align-content-flex-start-tablet {
+	@include tablet {
+		align-content: flex-start !important;
+	}
+}
+
+.align-content-flex-end-tablet {
+	@include tablet {
+		align-content: flex-end !important;
+	}
+}
+
+.align-content-center-tablet {
+	@include tablet {
+		align-content: center !important;
+	}
+}
+
+.align-content-stretch-tablet {
+	@include tablet {
+		align-content: stretch !important;
+	}
+}
+
+.align-items-stretch-tablet-tablet {
+	@include tablet {
+		align-items: stretch !important;
+	}
+}
+
+.align-items-flex-start-tablet {
+	@include tablet {
+		align-items: flex-start !important;
+	}
+}
+
+.align-items-flex-end-tablet {
+	@include tablet {
+		align-items: flex-end !important;
+	}
+}
+
+.align-items-baseline-tablet {
+	@include tablet {
+		align-items: baseline !important;
+	}
+}
+
+.align-self-flex-start-tablet {
+	@include tablet {
+		align-self: flex-start !important;
+	}
+}
+.align-self-flex-end-tablet {
+	@include tablet {
+		align-self: flex-end !important;
+	}
+}
+
+.align-self-center-tablet {
+	@include tablet {
+		align-self: center !important;
+	}
+}
+
+.align-self-baseline-tablet {
+	@include tablet {
+		align-self: baseline !important;
+	}
+}
+
+.align-self-stretch-tablet {
+	@include tablet {
+		align-self: stretch !important;
+	}
+}
+
+.flex-direction-row-tablet {
+	@include tablet {
+		flex-direction: row !important;
+	}
+}
+
+.flex-direction-row-reverse-tablet {
+	@include tablet {
+		flex-direction: row-reverse !important;
+	}
+}
+
+.flex-direction-column-tablet {
+	@include tablet {
+		flex-direction: column !important;
+	}
+}
+
+.flex-direction-column-reverse-tablet {
+	@include tablet {
+		flex-direction: column-reverse !important;
+	}
+}
+
+.justify-content-start-tablet {
+	@include tablet {
+		justify-content: flex-start !important;
+	}
+}
+
+.justify-content-space-between-tablet {
+	@include tablet {
+		justify-content: space-between !important;
+	}
+}
+
+.justify-content-space-around-tablet {
+	@include tablet {
+		justify-content: space-around !important;
+	}
+}
+
+.justify-content-space-evenly-tablet {
+	@include tablet {
+		justify-content: space-evenly !important;
+	}
+}
+
+// Other suggested Useful responsive classes for Desktop
+
+.align-content-flex-start-desktop {
+	@include desktop {
+		align-content: flex-start !important;
+	}
+}
```

Comment:
To make it easier to read the file, I'll recommend combining all mobile, tablet, and desktop rules together that belong to the same class. What I mean is instead of having these rules spread out in the file:

```
// line 3
.align-content-flex-start {
	align-content: flex-start !important;
}

// line 173
.align-content-flex-start-tablet {
	@include tablet {
		align-content: flex-start !important;
	}
}

// line 300
.align-content-flex-start-desktop {
	@include desktop {
		align-content: flex-start !important;
	}
}
```

Let's combine it like that:
```
.align-content-flex-start {
	align-content: flex-start !important;

	&-tablet {
		@include tablet {
			align-content: flex-start !important;
		}
	}

	&-desktop {
		@include desktop {
			align-content: flex-start !important;
		}
	}
}
```

Reply:
Same situation , Not going forward with the responsiveness .